tests/functional/Hydro/Noh/Noh-planar-1d.py

```diff
@@ -169,6 +174,7 @@
             restartStep = 10000,
             dataDirBase = "dumps-planar-Noh",
             restartBaseName = "Noh-planar-1d",
+            restartFileConstructor = SiloFileIO,
```

Comment:
That adds a command line argument that defaults to SiloFileIO, which is how it used to be and what is set in our controller. Then for the couple SidreFileIO ATS tests I specify that we want to test SidreFileIO when listing the tests at the top of the file.

Reply:
Not sure why I read Sidre instead of Silo, oops, but the same question would have applied. If this is mutable with the command line option it looks good to me. :+1: